src/quo/components/wallet/account_card/properties.cljs

```diff
@@ -0,0 +1,20 @@
+(ns quo.components.wallet.account-card.properties
```

Comment:
What is this properties file? Why are not these in `style` ?

Reply:
Probably because the values are not being used in style maps directly (section `Don't define properties in styles ns` in the guidelines). To me, they shouldn't be in the `style` ns, and could be directly on the same `view` file or in the properties file as @mmilad75 did.